GVFS/GVFS.FunctionalTests/FileSystemRunners/BashRunner.cs

```diff
@@ -233,7 +239,14 @@ public override void DeleteFile_FileShouldNotBeFound(string path)
 
         public override void DeleteFile_AccessShouldBeDenied(string path)
         {
-            this.DeleteFile(path).ShouldContain(permissionDeniedMessage);
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                this.DeleteFile(path).ShouldContain(permissionDeniedMessage);

```

Comment:
Alternatively, don't bother checking the platform, and just make this `ShouldContainOneOf(both messages)`

Reply:
I'd rather not relax the check when we're running the tests on Windows.  I like the idea of separate Windows\Mac BashRunners, but I also agree at this point we probably don't need both.